namespaces/live.cloud-platform.service.justice.gov.uk/offender-case-notes-prod/resources/rds.tf

```diff
@@ -11,9 +11,10 @@ module "dps_rds" {
   db_instance_class         = "db.t4g.medium"
   deletion_protection       = true
   prepare_for_major_upgrade = false
```

Comment:
Can you set the flag to true as per the guide: https://user-guide.cloud-platform.service.justice.gov.uk/documentation/deploying-an-app/relational-databases/upgrade.html#upgrading-to-a-new-major-database-version